### PR TITLE
Add entry for Cesium 3d Tiles to 3.34 changelog

### PR DESCRIPTION
**Pull request by nyalldawson**
I can't work out how to embed a video here, but the video at https://youtu.be/lvl8zVZ8glY should be attached to this entry

### CODE REVIEW

**Comment by DelazJ** [comment]
@nyalldawson example at line 16 doesn't work?

**Comment by nyalldawson** [comment]
@DelazJ Ah, I'll try, thanks!

**Comment by rduivenvoorde** [comment]
I'll be in the train to Vienna this evening, and will try to do this (in case this issue is still here).
(I'll also add the splash screen to it https://github.com/qgis/QGIS-Website/issues/1198)

File: source/site/forusers/visualchangelog334/index.rst [hunk 1]
```diff
@@ -21,6 +21,11 @@ QGIS is supported by donors and sustaining members. A current list of donors who
 
 QGIS is free software and you are under no obligation to pay anything to use it - in fact, we want to encourage people far and wide to use it regardless of their financial or social status - we believe that empowering people with spatial decision-making tools will result in a better society for all of humanity.
 
+Cesium 3D Tiles
+---------------
+
+Thanks to an ecosystem grant from the [Cesium](https://www.cesium.com/) project, QGIS now has support for loading in 3D content in the Cesium 3D Tiles format! Tiles can be loaded from either local or remote datasources (including the Cesium ion platform and the Google Earth photorealistic tiles), and are visible in both 2D and 3D map views. In 2D views users can control the appearance of tile content by styling in either their original textures or in a wireframe view of the tile geometries, and users also have control over the desired level of detail in both 3D and 2D views.
```
Comment (DelazJ) [inline source/site/forusers/visualchangelog334/index.rst:27]:
```suggestion
Thanks to an ecosystem grant from the `Cesium <https://www.cesium.com/>`_ project, QGIS now has support for loading in 3D content in the Cesium 3D Tiles format! Tiles can be loaded from either local or remote datasources (including the Cesium ion platform and the Google Earth photorealistic tiles), and are visible in both 2D and 3D map views. In 2D views users can control the appearance of tile content by styling in either their original textures or in a wireframe view of the tile geometries, and users also have control over the desired level of detail in both 3D and 2D views.
```

**Comment by rduivenvoorde** [comment]
I've added Harrissou's comment, added the Youtube video and gave the attribution/credits to Martin and Nyall.

See: https://github.com/qgis/QGIS-Website/commit/fb777ede06b54d3cff599db44c2e6e6791311c49

So closing this one, please fix if needed.

**Comment by nyalldawson** [comment]
Thanks @rduivenvoorde !!

**Pull request closed without merging**